Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/MicrosoftDefenderAdvancedThreatProtection.py

```diff
@@ -3385,42 +3385,98 @@ def get_machine_data(machine):
     return machine_data
 
 
-def get_file_statistics_command(client: MsClient, args: dict):
+def get_file_statistics_command(client: MsClient, args: dict) -> CommandResults:
     """Retrieves the statistics on the given file.
 
     Returns:
-        (str, dict, dict). Human readable, context, raw response
+        CommandResults.
     """
-    file_sha1 = args.get('file_hash')
+    file_sha1 = args.get('file_hash', '')
     response = client.get_file_statistics(file_sha1)
     file_stat = get_file_statistics_context(response)
-    human_readable = tableToMarkdown(f'Statistics on {file_sha1} file:', file_stat, removeNull=True)
     context_output = {
         'Sha1': file_sha1,
         'Statistics': file_stat
     }
-    entry_context = {
-        'MicrosoftATP.FileStatistics(val.Sha1 === obj.Sha1)': context_output
+    human_readable = get_file_statistics_human_readable(file_sha1, response)
+    file_indicator = get_file_statistics_indicator(file_sha1, response)
+
+    return CommandResults(
+        outputs_prefix='MicrosoftATP.FileStatistics',
+        outputs_key_field='Sha1',
+        indicator=file_indicator,
+        readable_output=human_readable,
+        outputs=context_output,
+        raw_response=response,
+    )
+
+
+def get_file_statistics_human_readable(file_hash: str, file_stat_response: dict) -> str:
+    """Gets the file statistics response and returns as a human readable markdown table.
+
+    Args:
+        file_hash (str): Hash of file
+        file_stat_response (dict): File statistics response body
+
+    Returns:
+        (str). File statistics human readable
+    """
+    formatted_data = {
+        'Global Prevalence': file_stat_response.get('globalPrevalence'),
+        'Organization Prevalence': file_stat_response.get('orgPrevalence'),
+        'Global First Observed': file_stat_response.get('globalFirstObserved'),
+        'Global Last Observed': file_stat_response.get('globalLastObserved'),
+        'Organization First Seen': file_stat_response.get('orgFirstSeen'),
+        'Organization Last Seen': file_stat_response.get('orgLastSeen'),
+        'Top File Names': file_stat_response.get('topFileNames'),
```

Comment:
all these `get`s are very simple, but are repeated both here, in `get_file_statistics_indicator` and in `get_file_statistics_context`. 
Can we create some object that will parse these files from the dictionary once, then create the suitable dictionaries (human readable, file stat context, file_stat indicator) instead?

Reply:
Fixed in commit e9f66b0abb115a5d3d2067356411fb235763b986 by creating a single dataclass object as we discussed in person